services/gitlab/gitlab-forks.service.js

```diff
@@ -0,0 +1,74 @@
+import Joi from 'joi'
+import { optionalUrl, nonNegativeInteger } from '../validators.js'
+import { metric } from '../text-formatters.js'
+import GitLabBase from './gitlab-base.js'
+
+const schema = Joi.object({
+  forks_count: nonNegativeInteger,
+}).required()
+
+const queryParamSchema = Joi.object({
+  gitlab_url: optionalUrl,
+}).required()
+
+const documentation = `
+<p>
+  You may use your GitLab Project Id (e.g. 278964) or your Project Path (e.g. gitlab-org/gitlab ).
+  Note that only internet-accessible GitLab instances are supported, for example https://jihulab.com, https://gitlab.gnome.org, or https://gitlab.com/.
+</p>
+`
+
+export default class GitlabForks extends GitLabBase {
+  static category = 'social'
+
+  static route = {
+    base: 'gitlab/forks',
+    pattern: ':project+',
+    queryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'GitLab forks',
+      namedParams: {
+        project: 'gitlab-org/gitlab',
+      },
+      queryParams: { gitlab_url: 'https://gitlab.com' },
+      staticPreview: {
+        label: 'Fork',
+        message: '6.4k',
+        style: 'social',
+      },
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = { label: 'forks', namedLogo: 'gitlab' }
+
+  static render({ baseUrl, project, forkCount }) {
+    return {
+      message: metric(forkCount),
+      color: 'blue',
+      link: [`${baseUrl}/${project}/-/forks/new`],
```

Comment:
The equivalent links for what we have on the GitHub badge would be

```js
[
    `${baseUrl}/${project}/-/forks/new`,
    `${baseUrl}/${project}/-/forks`
]
```

how about we do the same here?

Reply:
I refer to the GitHub Service code at https://github.com/badges/shields/blob/c64bb7e1f35c71d7679a415748b093c5e06badae/services/github/github-stars.service.js#L49-L50

Reply:
so the corresponding code for the equivalent github badge would be

https://github.com/badges/shields/blob/3f63990049b0f5967811aaced4c7db91a8bb2e51/services/github/github-forks.service.js#L49-L52

that makes the left side of the bade a link to for the repo and the right side a link to view forks. I'm suggesting we do the same for gitlab

Reply:
got it